### ISSUE TITLE
Test for WCAG success criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#173]
An alternative for time-based media or audio description of the prerecorded video content is provided for synchronized media, except when the media is a media alternative for text and is clearly labeled as such.
## Scope selector

Video content
## Applicable

synchronized media
## Not applicable
- The media is a media alternative for text and is clearly labeled as such.
- if all of the information in the video track is already provided in the audio track
## Relationship with other success criteria
- At Level A in Success Criterion 1.2.3, authors do have the choice of providing either an audio description or a full text alternative. 
- At level AA Under Success Criterion 1.2.5 authors must provide an audio description - a requirement already met if they chose that alternative for 1.2.3, otherwise an additional requirement. 
- At Level AAA under Success Criterion 1.2.8 they must provide an extended text description. This is an additional requirement if both 1.2.3 and 1.2.5 were met by providing an audio description only. If 1.2.3 was met, however, by providing a text description, and the 1.2.5 requirement for an audio description was met, then 1.2.8 does not add new requirements.
## Failures
- no failures defined
## Techniques

As we try to conform to AA level, we will first focus on 1.2.5, if 1.2.5 passes, this succes criterion passes, if not, it's a can't tell. If no video content is available, it's a n/a
